cypress-custom/integration/fee.test.ts

```diff
@@ -20,12 +20,35 @@ describe('Fee endpoint', () => {
 describe('Fetch and persist fee', () => {
   beforeEach(() => {
     cy.visit('/swap')
+
+    // Alias localStorage
+    cy.window()
+      .then(window => window.localStorage)
+      .as('localStorage')
   })
 
   it('Persisted when selecting a token', () => {
+    const TOKEN = 'ETH'
     // GIVEN: A fee for a token is not in the local storage
     // WHEN: When the user select this token
+
+    // Clear localStorage
+    cy.get<Storage>('@localStorage').then(storage => storage.clear())
+
     // THEN: The fee is persisted in the local storage (redux_localstorage_simple_fee)
+    cy.get<Storage>('@localStorage')
+      .its('redux_localstorage_simple_fee')
+      .should($feeStorage => {
+        // we need to parse JSON
+        const feeStorage = JSON.parse($feeStorage)
+        expect(feeStorage).to.have.property('4')
+        expect(feeStorage[4]).to.have.property(TOKEN)
+        expect(feeStorage[4][TOKEN]).to.have.property('token')
```

Comment:
can u create a const like `const feeQuote = feeStorage[ChainId.Rinkeby][TOKEN]` to avoid repetition?

Reply:
ok, this check was on purpose tho, to make sure each property exists but i can simplifiy it